app/collins/models/LshwHelper.scala

```diff
@@ -86,6 +92,41 @@ object LshwHelper extends CommonHelper[LshwRepresentation] {
     )
   }
 
+  protected def reconstructGpu(meta: Map[Int, Seq[MetaWrapper]]): FilteredSeq[Gpu] = {
```

Comment:
Will do, I didn't change this with my most recent commit (d019c27fb30c1541dbb31de075a834283154060a) but I'll get this cleaned up.